azurelinuxagent/ga/exthandlers.py

```diff
@@ -652,6 +674,8 @@ def __init__(self, ext_handler, protocol):
         self.is_upgrade = False
         self.logger = None
         self.set_logger()
+        self.handler_env = {EXTENSION_PATH: self.get_base_dir(),
```

Comment:
Added a new property in ExtHandlerInstance to maintain a common set of Environment Variables that would be passed to every launch_command call

Reply:
What's the purpose of these environment variables? Maybe I'm missing something, but I don't see them being used anywhere?

Reply:
This was an ask by the publishers to provide them with Extension_path and Extension_Version on every call to the extension so that it would save them the time to parse it.